doc/rados/operations/pools.rst

```diff
@@ -378,45 +381,41 @@ You may set values for the following keys:
 
 .. describe:: pg_num
 
-   The effective number of placement groups to use when calculating
+   Sets the effective number of PGs to use when calculating
    data placement.
 
    :Type: Integer
-   :Valid Range: Superior to ``pg_num`` current value.
+   :Valid Range: Greater than the current value of ``pg_num``.
```

Comment:
How has this been this wrong so long?

>The number of PGs to belong to this pool.

:Valid Range: 0 to `mon_max_pool_pg_num`.  If set to 0, the value of `osd_pool_default_pg_num` will be used.

https://github.com/ceph/ceph/blob/b554518133a6fa527cc6b02d7fcf4fb7410e1f0d/src/mon/OSDMonitor.cc#L7958

Reply:
Accepted. Good research.

And how could it have been wrong for so long? There are so few of us, and so many lines to check.

Reply:
I'm embarrassed that *I* didn't notice this before as an operator.